cogs/obliterate.py

```diff
@@ -254,6 +255,68 @@ async def on_error(self, interaction: discord.Interaction, error: Exception):
         print(error)
         traceback.print_tb(error.__traceback__)
 
+class AppreciationModal(discord.ui.Modal, title="Message of Appreciation"):
+    def __init__(self, bot, appreciate_players):
+        super().__init__()
+        self.bot = bot
+        self.appreciate_player = appreciate_players
+
+    message = discord.ui.TextInput(label='Message of appreciation', min_length=1, max_length=200, required=True, style=TextStyle.short)
+    anonymous = discord.ui.TextInput(label='Would you like this post to be anonymous?', min_length=1, max_length=3, required=True, placeholder="Yes/No", style=TextStyle.short)
+
+    async def on_submit(self, interaction: discord.Interaction):
+        user_id = int(interaction.message.embeds[0].footer.text.replace('User ID: ', ''))
+        member = await interaction.guild.fetch_member(user_id)
+        message = self.message.value
+        anonymous = self.anonymous.value
+        appreciate_players = self.appreciate_players
+
+        if not anonymous.upper() in ['NO', 'N', 'YES', 'Y']:
```

Comment:
Slightly more generic: `if not 'Y' in anonymous.upper() and not 'N' in anonymous.upper():`